Add packaging and query APIs for app packages (#160)
terrafx/terrafx.interop.windows: generation/other/mssign32/header.txt

// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.

// Ported from https://docs.microsoft.com/en-us/windows/win32/appxpkg/how-to-programmatically-sign-a-package

> // Original source is Copyright © Microsoft. All rights reserved. License information available at https://github.com/MicrosoftDocs/win32/blob/docs/LICENSE-CODE

Reviewer: I'd prefer this explicitly state the license at the time it was ported, rather than relying on a potentially updatable url:
```suggestion
// Original source is Copyright © Microsoft. All rights reserved. Licensed under the MIT License (MIT).
```

Reviewer: Same for the other files, changes otherwise LGTM.
